package/src/components/Attachment/Gallery.tsx

```diff
@@ -605,6 +605,7 @@ export const Gallery = (props: GalleryProps) => {
   const videos = propVideos || contextVideos;
   const message = propMessage || contextMessage;
 
+  console.log('TEST: ', images, videos);
```

Comment:
We should remove it. Must have appeared mistakenly

Reply:
Thanks !